Testing/functional/tests/cypress/integration/ui/errors/tooManyRequestsAlerts.js

```diff
@@ -0,0 +1,495 @@
+        cy.get(collectionDateYearSelector).select(collectionDateYear);
+        cy.get(collectionDateMonthSelector).select(collectionDateMonth);
+        cy.get(collectionDateDaySelector).select(collectionDateDay);
+        cy.get("[data-testid=btnEnter]")
+            .should("be.enabled", "be.visible")
+            .click();
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Public Vaccine Card Form", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.enableModules(["Immunization", "VaccinationStatus"]);
+        cy.logout();
+        cy.visit(vaccineCardUrl);
+
+        cy.intercept("GET", "**/PublicVaccineStatus", {
+            statusCode: 429,
+        });
+
+        enterVaccineCardPHN(fullyVaccinatedPhn);
+        cy.get(dobYearSelector).select(fullyVaccinatedDobYear);
+        cy.get(dobMonthSelector).select(fullyVaccinatedDobMonth);
+        cy.get(dobDaySelector).select(fullyVaccinatedDobDay);
+        cy.get(dovYearSelector).select(fullyVaccinatedDovYear);
+        cy.get(dovMonthSelector).select(fullyVaccinatedDovMonth);
+        cy.get(dovDaySelector).select(fullyVaccinatedDovDay);
+        clickVaccineCardEnterButton();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Public Vaccine Card Downloads", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.enableModules([
+            "Immunization",
+            "VaccinationStatus",
+            "VaccinationStatusPdf",
+            "PublicVaccineDownloadPdf",
+        ]);
+        cy.logout();
+        cy.intercept("GET", "**/PublicVaccineStatus", {
+            fixture: "ImmunizationService/publicVaccineStatusLoaded.json",
+        });
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(fullyVaccinatedPhn);
+        cy.get(dobYearSelector).select(fullyVaccinatedDobYear);
+        cy.get(dobMonthSelector).select(fullyVaccinatedDobMonth);
+        cy.get(dobDaySelector).select(fullyVaccinatedDobDay);
+        cy.get(dovYearSelector).select(fullyVaccinatedDovYear);
+        cy.get(dovMonthSelector).select(fullyVaccinatedDovMonth);
+        cy.get(dovDaySelector).select(fullyVaccinatedDovDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.intercept("GET", "**/PublicVaccineStatus/pdf", {
+            statusCode: 429,
+        });
+        cy.get("[data-testid=save-dropdown-btn]")
+            .should("be.visible", "be.enabled")
+            .click();
+        cy.get("[data-testid=save-as-pdf-dropdown-item]")
+            .should("be.visible")
+            .click();
+        cy.get("[data-testid=genericMessageModal]").should("be.visible");
+        cy.get("[data-testid=genericMessageSubmitBtn]").click();
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+        cy.get("[data-testid=loadingSpinner]").should("not.be.visible");
```

Comment:
Do we need this check for 'loadingSpinner'? Depending on why this check is being done, 'not.exist' might be safer to use than 'not.be.visible'.

Reply:
Okay I didn't realize those extra checks would cause flakiness so I removed all the checks except for the 'too many requests' alerts that we are actually testing here.  I also noticed some extra duplicate code that wasn't needed for the tests so cleaned it up. 
As for the loading spinner, I just did the same as what our current tests do so we may want to revisit that in those tests if it is a problem.

Reply:
@BrianMaki @rayking29 What to check depends on whether the element is hidden using v-if or v-show.

| Hiding Method | Presence | Non-Presence |
| :--- | :--- | :--- |
| v-if | "exist" or "be.visible" | "not.exist" |
| v-show | "be.visible" | "not.be.visible" |

Reply:
@MikeLyttle @BrianMaki I checked the LoadingComponent.vue that has the loadingSpinner test id, and the template div uses v-show="isLoading".
<template>
    <div
        v-show="isLoading"
        class="vld-parent"
        :class="isCustom && fullScreen ? 'fullScreen' : 'block'"
    >
So it looks like the "not.be.visible" is the correct check for this.